new-lamassu-admin/src/components/date-range-picker/Calendar.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import moment from 'moment'
+import { toInteger } from 'lodash/fp'
+import { makeStyles } from '@material-ui/core/styles'
+import Tile from './Tile'
+import { ReactComponent as Arrow } from '../../styling/icons/arrow/month_change.svg'
+import { primaryColor, zircon, fontSecondary } from '../../styling/variables'
+import typographyStyles from '../typography/styles'
+const { label2 } = typographyStyles
```

Comment:
We should stablish a styleguide for imports organization. I like to do it this way, separating types with a whitespace.

```js
# imports from node-modules
import React, { useState } from 'react'
import moment from 'moment'
import { toInteger } from 'lodash/fp'
import { makeStyles } from '@material-ui/core/styles'

# imports from components outside of ./
import { ReactComponent as Arrow } from '../../styling/icons/arrow/month_change.svg'
import { primaryColor, zircon, fontSecondary } from '../../styling/variables'
import typographyStyles from '../typography/styles'

# imports from this components folder
import Tile from './Tile'

# not imports
const { label2 } = typographyStyles
```